packages/app-tech-comm/src/useCounter.ts

```diff
@@ -9,7 +9,7 @@ import { useApi, trackStream } from '@polkadot/react-hooks';
 
 export default function useCounter (): number {
   const { api, isApiReady } = useApi();
-  const proposals = trackStream<Hash[]>(isApiReady ? api.query.technicalCommittee.proposals : undefined, []);
+  const proposals = trackStream<Hash[]>(isApiReady ? api.query.technicalCommittee && api.query.technicalCommittee.proposals : undefined, []);
```

Comment:
```suggestion
  const proposals = trackStream<Hash[]>(isApiReady ? api.query.technicalCommittee?.technicalCommittee.proposals : undefined, []);
```

Reply:
I think the suggested change here should be the following:
```
api.query.technicalCommittee?.proposals
```

Instead of:
```
api.query.technicalCommittee?.technicalCommittee.proposals
```

Otherwise I still get error `Uncaught TypeError: Cannot read property proposals of undefined`

Reply:
Little edit boxes :)